### ISSUE TITLE
Platte

### ISSUE THREAD

**Issue by charligel** [MakeMagazinDE/Nistkasten-V2#3]
Kann es sein, dass die Aussparungen für die Helligkeitssenoren auf der 3D-Druck-Platte mit Abstand Deckeö verkehrt angebracht sind und damit eine Montage nicht möglich ist?

**Comment by steffenknott** [comment]
Ja, das kann ich bestätigen. Ich habe versuchsweise im Slicer das Modell gespiegelt und neu gedruckt, das scheint zu funktionieren.

**Comment by schaeuzerle** [comment]
Ja, ist mir auch aufgefallen, nachdem ich es gedruckt habe... Hab einen Fork erstellt, darin liegt nun ein gespiegeltes STL, leicht verschobener Kameraposition und Halterung für einen Raspberry 3B, wenn jemand Interesse hat

**Comment by Lumixnick** [comment]
Hallo ist mir auch aufgefallen und ich hab sie kurzerhand umdesigned.
Hier meine STL von gerade. Hoffe ich hab mich nirgends vermessen, meine RasPI liegt auch einfach flach auf, benötige keinen USB-HUB. 
[Nistkasten v2_gespiegelt.zip](https://github.com/MakeMagazinDE/Nistkasten-V2/files/6063810/Nistkasten.v2_gespiegelt.zip)

**Comment by cyrix2000** [comment]
Ich habe eine neue stl Datei erstellt und einen pull request gestartet.
